suricata/update/main.py

```diff
@@ -387,10 +387,22 @@ def write_merged(filename, rulemap):
                         len(added),
                         len(removed),
                         len(modified)))
-    
     with io.open(filename, encoding="utf-8", mode="w") as fileobj:
-        for rule in rulemap:
-            print(rulemap[rule].format(), file=fileobj)
+        for sid in rulemap:
+            rule = rulemap[sid]
+            if "dataset" in rule:
+                load_attr = [el for el in rule.dataset.split(",") if "load" in el][0]
+                dataset_fname = load_attr.split(" ")[1] # ASK if there can only be one filename
```

Comment:
Could there be just one filename with load param?

Reply:
Yes, I think its OK to assume this.